README.md

```diff
@@ -1,65 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
+> A simple project setup  with html/css with support for linters and web accessibility.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./screenshot.png)
```

Comment:
- [ ] Your screenshot is currently not showing because it is referring to a file that does not exist.  Kindly consider changing the file path/name here.